specification/search/resource-manager/Microsoft.Search/stable/2023-11-01/search.json

```diff
@@ -2411,6 +2562,20 @@
       "required": true,
       "type": "string",
       "description": "The API version to use for each request."
+    },
+    "LocationParameter": {
+      "name": "location",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The unique location name for a Microsoft Azure geographic region."
+    },
+    "SkuNameParameter": {
+      "name": "skuName",
+      "in": "path",
+      "required": true,
+      "type": "string",
```

Comment:
"$ref": "#/definitions/Sku"?

Reply:
This is a query parameter, it wouldn't reference anything else.

Reply:
is there a way to explicitly define the values of the parameter?

Reply:
I am not sure of this, but I have added a check in the code to default for invalid skuName inputs. It would be great to have a fixed set of defined values for this parameter though. Please let me know.